src/py/flwr/cli/utils.py

```diff
@@ -161,10 +161,43 @@ def get_sha256_hash(file_path: Path) -> str:
 
 
 def get_user_auth_config_path(root_dir: Path, federation: str) -> Path:
-    """Return the path to the user auth config file."""
+    """Return the path to the user auth config file.
+
+    Additionally, a `.gitignore` file will be created or updated in the
+    Flower directory to include `.credentials`."""
     # Locate the credentials directory
     credentials_dir = root_dir.absolute() / FLWR_DIR / CREDENTIALS_DIR
     credentials_dir.mkdir(parents=True, exist_ok=True)
+
+    # Determine the absolute path of the Flower directory for .gitignore
+    abs_flwr_dir = root_dir.absolute() / FLWR_DIR
```

Comment:
should we make use of this variable earlier in the function? to avoid mistakes? (`credentials_dir` shares part of this)